src/core.js

```diff
@@ -30,7 +30,7 @@ define([
     'jquery.jstree',
     'jquery.bootstrap',
     'jquery.fancybox',  // only thing we use fancybox for is its spinner, no actual display of anything
-    'jquery-ui',  // used for buttons in Edit Source XML, and dialogs
+    'jquery-ui',  // used for autocomplete
```

Comment:
is it going to be possible to substitute the autocomplete functionality as well, or are we stuck with jquery-ui?